.travis.yml

```diff
@@ -4,4 +4,9 @@ node_js:
 env:
   - TEST_DIR: .
   - TEST_DIR: extension
+
+sudo: required
```

Comment:
todo? according to the other issue it seems like this is only a temporary workaround.